Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/server/VerificationKeyType.kt

```diff
@@ -1,5 +1,11 @@
 package de.rki.coronawarnapp.coronatest.server
 
+import com.google.gson.annotations.SerializedName
+
 enum class VerificationKeyType {
-    GUID, TELETAN;
+    @SerializedName("GUID")
+    GUID,
+
+    @SerializedName("TELETAN")
+    TELETAN;
```

Comment:
We previously just relied on the enums never being renamed or obfuscated, this was a bit too fragile for my taste.